defaults/main.yml

```diff
@@ -31,3 +33,17 @@ couchdb_pki_cert_suffix: '.pem'
 
 couchdb_init: systemd
 couchdb_activate: True
+couchdb_async_io_threads: 16
```

Comment:
if u find it to be a useful knob in your build/deployment then add it. if not having it present means u get the default, then I would add it commented to the defaults, and with an {% if foo is defined %}foo = {{foo}}{% else %}#foo = 16{% endif %} so that you can opt into your value iff it's specified.